etna/datasets/utils.py

```diff
@@ -0,0 +1,92 @@
+from enum import Enum
+from typing import Sequence
+
+import pandas as pd
+
+from etna.datasets.tsdataset import TSDataset
+
+
+class DataFrameFormat(str, Enum):
+    """Enum for different types of result."""
+
+    wide = "wide"
+    long = "long"
+
+
+def duplicate_data(df: pd.DataFrame, segments: Sequence[str], format: str = DataFrameFormat.long) -> pd.DataFrame:
```

Comment:
I would choose wide format by default, because it is etna format. But it's is just my opinion.

Reply:
Fixed